app/models/magic.py

```diff
@@ -0,0 +1,31 @@
+import time
+from dataclasses import dataclass
+
+
+@dataclass
+class Magic:
+    name: str
+    base_power: int
+
+    def get_power(self) -> int:
+        """
+        Обчислює магічну силу для Ланселота в залежності від поточного часу.
+
+        Часові діапазони:
+        - З 12:00 по 17:59: +15 до сили.
+        - З 18:00 по 23:59: 0 до сили.
+        - З 00:00 по 05:59: -10 до сили.
+        - З 06:00 по 11:59: 0 до сили.
+
+        Повертає значення магічної сили.
+        """
```

Comment:
The docstring is not written in English. It's a good practice to write comments and docstrings in English, as it's the universal language of programming. This makes your code more understandable to a wider audience of developers.